api/src/main/java/org/openmrs/module/m2sysbiometrics/service/RegistrationService.java

```diff
@@ -0,0 +1,13 @@
+package org.openmrs.module.m2sysbiometrics.service;
+
+import org.openmrs.module.m2sysbiometrics.bioplugin.LocalBioServerClient;
+import org.openmrs.module.m2sysbiometrics.bioplugin.NationalBioServerClient;
+import org.openmrs.module.m2sysbiometrics.model.M2SysCaptureResponse;
+import org.openmrs.module.registrationcore.api.biometrics.model.BiometricSubject;
+
+public interface RegistrationService {
+
+    void register(LocalBioServerClient client, BiometricSubject subject, M2SysCaptureResponse capture);
```

Comment:
can't we just inject the servers to the bean and remove them from the interface methods?

Reply:
ok I will do it